src/printer/layers.js

```diff
@@ -128,9 +128,13 @@ function createTiledLayer(source, rootFrameState, opacity) {
           { ...frameState, time: Date.now() },
           context.canvas
         );
-        return [1, context.canvas, tileLoadErrorUrl];
+        return tileLoadErrorUrl
```

Comment:
I d'ont understand why you have to do this

Reply:
To make `tileLoadErrorUrl` optional in layerstate. The alternative would be to always include `tileLoadErrorUrl` (also when it's `undefined` as it was the case before https://github.com/camptocamp/inkmap/pull/16/commits/b4bbc3d8c0a7d4b9a881cd56f711d5345efabb33)

Reply:
Ok cancel the https://github.com/camptocamp/inkmap/commit/b4bbc3d8c0a7d4b9a881cd56f711d5345efabb33 I prefer as it was before, more explicit.

BTW this line could be refactored `return [progress, context.canvas, tileLoadErrorUrl]` is true in both case (also when `progress === 1`)